tests/api_fastapi/views/ui/test_assets.py

```diff
@@ -34,14 +34,14 @@ def cleanup():
     initial_db_init()
 
 
-def test_next_run_datasets(test_client, dag_maker):
+def test_next_run_assets(test_client, dag_maker):
```

Comment:
Same comment about dataset_expression -> asset_expression here

Reply:
This is also a db column in DAG model. I intend to change it in https://github.com/apache/airflow/pull/42023